test/parallel/test-crypto-dh.js

```diff
@@ -61,6 +61,12 @@ for (const bits of [-1, 0, 1]) {
   crypto.createDiffieHellman('abcdef', 'hex', g);
 }
 
+assert.throws(() => crypto.createDiffieHellman('', 'base64', () => {}), {
+  code: 'ERR_OSSL_BN_BITS_TOO_SMALL',
+  name: 'Error',
+  message: /bits too small/,
+});
+
```

Comment:
Can you put this at the end of the file please
```suggestion
[true, Symbol(), {}, () => {}, []].forEach((generator) => assert.throws(
  () => crypto.createDiffieHellman('', 'base64', generator),
  { code: 'ERR_INVALID_ARG_TYPE' }
));
```